data/fibrousfinance.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Fibrous Finance",
+  "description": "Fibrous is a price exploration and routing algorithm across multiple AMMs that offers token exchanges with the best and cheapest rate at StarkNet validity Rollup.",
+  "short_description": "Gathers all AMMs of StarkNet at one place.",
+  "tags": [
+    "DeFi"
+  ], 
+  "contracts": [],
+  "goerliContracts": [
+    {
+      "name": "Router Contract",
+      "address": "0x7c6f04a78f405bb2ff08cb07a1a1e0868800fa1ed41d9067d8f8f28ba73a4f9"
+    }
+  ],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://fibrous.finance/",
+    "careers": "",
+    "twitter": "https://twitter.com/FibrousFinance",
+    "telegram": "",
+    "discord": "",
+    "github": "https://github.com/Fibrous-Finance",
+    "youtube": "",
+    "medium": "https://medium.com/@fibrous",
+    "mirror": ""
+  },
+  "twitterName": "FibrousFinance",
+  "teamInfo": {
+    "founded": "2022-04-01T00:00:00.000Z",
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://i.hizliresim.com/5ne373k.jpg",
+    "bannerUrl": "https://i.hizliresim.com/r5uy2xt.png",
```

Comment:
It looks like this at the moment:

<img width="1912" alt="Screenshot 2022-10-06 at 16 53 39" src="https://user-images.githubusercontent.com/10310633/194346479-30904417-8257-4df5-9be2-a7de42f14eec.png">

Not sure if you guys are happy with the way the banner image looks like (pixelated, a bit zoomed in)

Reply:
I fixed it and sent it